Assembly reference missing with relative path to .csx

Running the nightly `scriptcs v0.5.0-alpha-130524` to get the ScriptArgs functionality (which works great :smile:)

When running (via powershell)

```
 & "scriptcs" "../Utilities/scriptcs/mysql/CreateNewMySQLUser.csx" -- -username $username -dbAdmin bob -pass (Read-Host -assecurestring "Please enter the password for mysql user bob")
```

I get `ERROR: (1,7): error CS0246: The type or namespace name 'MySql' could not be found (are you missing a using directive or an assembly reference?)`

Where as when running

```
 & "scriptcs" ./CreateNewMySQLUser.csx -- -username $username -dbAdmin bob -pass (Read-Host -assecurestring "Please enter the password for mysql user bob")
```

It works as expected.

Not a major issue as I can change directory in my script, but probably not desired behaviour!

I could of course be completely wrong!

--- a/src/ScriptCs.Core/FileSystem.cs
+++ b/src/ScriptCs.Core/FileSystem.cs
@@ -90,7 +90,12 @@ namespace ScriptCs
         {
             var realPath = GetFullPath(path);
 
-            return Path.GetFileName(realPath) == string.Empty ? realPath : Path.GetDirectoryName(realPath);
+            var attributes = File.GetAttributes(realPath);
+
+            if ((attributes & FileAttributes.Directory) == FileAttributes.Directory)
+                return realPath;
+            else
+                return Path.GetDirectoryName(realPath);
         }
 
         public string GetFullPath(string path)

--- a/test/ScriptCs.Core.Tests/ScriptExecutorTests.cs
+++ b/test/ScriptCs.Core.Tests/ScriptExecutorTests.cs
@@ -158,9 +158,20 @@ namespace ScriptCs.Tests
             [Fact]
             public void ShouldResolveAssemblyReferencesCorrectlyWhenNotRunFromCSXFolder()
             {
-                var fileSystem = new FileSystem();
+                const string pathToMyScriptFolder = @"..\my_script\";
 
-                Assert.Equal(Path.GetFullPath(Path.Combine(Environment.CurrentDirectory,@"..\my_script\")), fileSystem.GetWorkingDirectory(@"..\my_script\"));
+                try {
+                    Directory.CreateDirectory(pathToMyScriptFolder);
+
+                    var fileSystem = new FileSystem();
+
+                    fileSystem.GetWorkingDirectory(pathToMyScriptFolder)
+                              .ShouldEqual(Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, pathToMyScriptFolder)));
+                }
+                finally {
+                    if (Directory.Exists(pathToMyScriptFolder))
+                        Directory.Delete(pathToMyScriptFolder);
+                }
             }
 
             [Fact]

